padrino-core/lib/padrino-core/cli/rake_tasks.rb

```diff
@@ -70,3 +62,11 @@ def env_migration_version
     list_app_routes(app, args) if app
   end
 end
+
+Dir["{lib/tasks/**,tasks/**,test,spec}/*.rake"].each do |file|
```

Comment:
Why even bother loading `test` and `spec` unless we're in the test environment? This alleviates the need to do [this](https://github.com/padrino/padrino-framework/pull/1010/files).